src/main/webapp/app/exercises/file-upload/participate/file-upload-submission.component.ts

```diff
@@ -119,10 +119,10 @@ export class FileUploadSubmissionComponent implements OnInit, ComponentCanDeacti
                     .join(',');
                 this.isAfterAssessmentDueDate = !this.fileUploadExercise.assessmentDueDate || dayjs().isAfter(this.fileUploadExercise.assessmentDueDate);
 
-                if (this.submission.submitted) {
+                if (this.submission?.submitted) {
                     this.setSubmittedFile();
                 }
-                if (this.submission.submitted && this.result && this.result.completionDate) {
+                if (this.submission?.submitted && this.result && this.result.completionDate) {
```

Comment:
```suggestion
                if (this.submission?.submitted && this.result?.completionDate) {
```
If you want, you can shorten this line